src/config/configStore.ts

```diff
@@ -219,22 +190,22 @@ export abstract class BaseConfigStore<
    *
    * @param keys The keys. Supports query keys like `a.b[0]`.
    */
-  public unsetAll(keys: string[]): boolean {
-    return keys.reduce((val: boolean, key) => val && this.unset(key), true);
+  public unsetAll(keys: Array<Key<P>>): boolean {
```

Comment:
The jsdoc mentions that it supports `a.b[0]` keys - is that still the case? I know the PR description mentioned that it won't work for `get` and `set`